### ISSUE TITLE
cannot acces properties menu, no menu in gui

### ISSUE THREAD

**Issue by fenpaws** [ricardomv/snapper-gui#43]
So I have a wired error in the latest SnapperGUI, 
I try to go to the properties section but I cannot open the menu for it. There should be a menu when you click the icon on the top left, or at least it was this way in xfce.

I could find anything that would suggest a settings page. So i think its a bug maybe.

**My system:**
OS: Manjaro 20.2 Nibia 
DE: KDE 5.76.0 / Plasma 5.20.4

It a fresh install btw.

**Comment by tazihad** [comment]
Yes, same problem. Clicking the icon doesn't bring any properties option.

**Comment by rickysarraf** [comment]
I guess this blame should be on Gtk. I'm on KDE with Global menu and I do get the menu exported there. But I'm not sure why the initial reported has the issue because he's on KDE.

My system is: Debian Unstable  + KDE 

![image](https://user-images.githubusercontent.com/65480/118009581-cc932500-b36b-11eb-9350-d0502b5c59f9.png)

**Comment by kaaass** [comment]
just meet the same problems

However, I found run snapper gui in root privilege (`sudo snapper-gui`) could fix this issue

**Comment by geckolinux** [comment]
@rickysarraf Hi, thanks very much for creating this extremely useful GUI.
I'm also having this issue on Debian Stable with Plasma. Under XFCE on Debian Stable the properties menu works correctly. I tried some custom Kwin rules too but no success.

**Comment by geckolinux** [comment]
Thanks for the reply. I suspect that it has something to do with the way Kwin tries to deal with CSD. I also tried it in Icewm on the same system and it acted the same way, but again CSD can behave weird with different window managers.

**Comment by ZerkerEOD** [comment]
I am having this issue also.
